src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -358,6 +360,7 @@ protected function createClientRequest($data, array $headers = null)
 
         $config->set('curl.options', $curlOptions);
         $this->httpClient->setConfig($config);
+        $this->httpClient->setDefaultOption('verify', __DIR__ . '/../../certs/cacert.pem');
```

Comment:
Rather than hard-coding this cert dependency and comiting the cert file manually what do you think about managing this via composer. For example with [composer/ca-bundle](https://github.com/composer/ca-bundle) and then you can use it with Guzzle like so https://github.com/composer/ca-bundle#to-use-with-guzzle

Reply:
oh that's awesome, good idea @JCManzo!

Reply:
also yay @DanayaMel for figuring out how to do this with guzzle functions!